src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,41 @@
+import { TTile } from '../../types';
+
+export function AppTile({
+    width,
+    height,
+    content,
+    textSize,
+    imgOptions,
+    onClick,
+}: TTile): HTMLElement {
+    const tile = document.createElement('div');
+    tile.className = `rounded-lg bg-tile-bg-gray flex`;
+    tile.classList.add(width, height);
+
+    if (content.startsWith('http')) {
```

Comment:
You can local images and this would not work, I know it's not the case in this project, but just overall better to have two props

Reply:
Fixed